source/server/getting-started/implementations/spongeforge.rst

```diff
@@ -23,33 +23,45 @@ However, SpongeForge usually updates to a new Forge build fairly soon after it's
 worry about always having to run an outdated Forge version in order to use SpongeForge.
 
 
-The format of the filename is ``spongeforge-{MCVersion}-{ForgeVersion}-{SpongeAPIVersion}-{SpongeBuildId}``
-
-+----------------------+----------------------------------------------------------------------------------------------+
-| ``MCVersion``        | The Minecraft version. Only clients compatible with this version can connect.                |
-+----------------------+----------------------------------------------------------------------------------------------+
-| ``ForgeVersion``     | The version of Forge this file is built for. Preferably your server should run this exact    |
-|                      | version of Forge.                                                                            |
-+----------------------+----------------------------------------------------------------------------------------------+
-| ``SpongeAPIVersion`` | The version of the SpongeAPI implemented by this file. This is what Sponge plugins depend on.|
-+----------------------+----------------------------------------------------------------------------------------------+
-| ``SpongeBuildId``    | The build number of Sponge. This is what you should supply when reporting bugs or seeking    |
-|                      | support.                                                                                     |
-+----------------------+----------------------------------------------------------------------------------------------+
+The format of the filename is ``spongeforge-<MCVersion>-<ForgeBuildId>-<SpongeAPIVersion>-BETA-<SpongeBuildId>.jar``
+
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``MCVersion``        | The Minecraft version. Only clients compatible with this version can connect.                 |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``ForgeBuildId``     | The build number of Forge this file is built for. Preferably your server should run this exact|
+|                      | version of Forge. Which is the last part of Forge's version string.                           |
```

Comment:
This should be one sentence, and made a bit clearer. ie. ``Preferably your server should run this exact version of Forge (which can be identified in the last part of Forge's version string)``.